### PR TITLE
Add DismissRecommendation example

### PR DESCRIPTION
**Pull request by nwbirnie**

### CODE REVIEW

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/recommendations/DismissRecommendation.java [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    if (!params.parseArguments(args)) {
+
+      // Either pass the required parameters for this example on the command line, or insert them
+      // into the code here. See the parameter class definition above for descriptions.
+      params.customerId = Long.parseLong("INSERT_CUSTOMER_ID_HERE");
+
+      // Recommendation ID is the last alphanumeric portion of the value from
+      // Recommendation.getResourceName(), which has the format of
+      // `customers/<customer_id>/recommendations/<recommendation_id>`.
+      // Its example can be retrieved from GetTextAdRecommendations.java
```
Comment (jradcliff) [inline google-ads-examples/src/main/java/com/google/ads/googleads/examples/recommendations/DismissRecommendation.java]: nit: add period.

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/recommendations/DismissRecommendation.java [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+  /**
+   * Runs the example.
+   */
+  private void runExample(GoogleAdsClient googleAdsClient, Long customerId,
+      String recommendationId) {
+    // Convert the customer and recommendation ID to a resource name.
+    String resourceName = ResourceNames.recommendation(customerId, recommendationId);
+
+    // Construct a dismiss operation for the recommendation.
```
Comment (jradcliff) [inline google-ads-examples/src/main/java/com/google/ads/googleads/examples/recommendations/DismissRecommendation.java]: nit: Let's use the form that implies a prefix of "This line...", as you did on line 105.

So 97 and 100 would be "Converts" and "Constructs", respectively.

Comment (nwbirnie) [reply google-ads-examples/src/main/java/com/google/ads/googleads/examples/recommendations/DismissRecommendation.java]: Done

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/recommendations/DismissRecommendation.java [hunk 3]
```diff
@@ -0,0 +1,116 @@
+// Copyright 2018 Google LLC
```
Comment (jradcliff) [inline google-ads-examples/src/main/java/com/google/ads/googleads/examples/recommendations/DismissRecommendation.java]: 2019

Comment (nwbirnie) [reply google-ads-examples/src/main/java/com/google/ads/googleads/examples/recommendations/DismissRecommendation.java]: Done

**Pull request merged**